examples/with-shell-commands/packages/with-workspace-configuration/turbo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["!README.md"],
```

Comment:
As soon as you have an `inputs` config, all other files in the repo are excluded so you have to add them back manually. I think this is a dangerous example to give